### ISSUE TITLE
Failed to run CC harvester: invalid character in identifier

### ISSUE THREAD

**Issue by mprostock** [tonybaloney/wily#64]
wily version 1.12.1, installed via pip in a clean venv with python 3.6.7 on ubuntu.

I am trying to run `wily build` on a project of mine. Unfortunately I cannot make it work. It finds an "invalid character" but there is none. See screenshot of relevant code piece below. Even deleting the file from the repo does not help because of course it is still in the git history (and I am not going to soft-reset through 30 commits to get wily to work). 

Even if there were an invalid character, couldn't there be a way to more gracefully handle this and simply exclude that file from the index/reporting?

Key error:
`Failed to run CC harvester on dashtry.py : invalid character in identifier (<unknown>, line 29)`
Following stack:
```Failed to build cache: ''str' object has no attribute 'h1''
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 47, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "/Data/Code/projects/finance_data/.venv/lib/python3.6/site-packages/wily/commands/build.py", line 24, in run_operator
    return operator.name, instance.run(revision, config)
  File "/Data/Code/projects/finance_data/.venv/lib/python3.6/site-packages/wily/operators/halstead.py", line 81, in run
    results[filename] = self._report_to_dict(instance)
  File "/Data/Code/projects/finance_data/.venv/lib/python3.6/site-packages/wily/operators/halstead.py", line 86, in _report_to_dict
    "h1": report.h1,
AttributeError: 'str' object has no attribute 'h1'
"""
```

![image](https://user-images.githubusercontent.com/12598309/54341410-2e478f80-463a-11e9-8942-abea78b2d373.png)

Running wily subsequently on a subdirectory leads to this, so apparently wily leaves "corrupted" files behind when exiting in the above fashion:

![image](https://user-images.githubusercontent.com/12598309/54342455-ccd4f000-463c-11e9-924c-a0236164291c.png)

output of pip freeze:
attrs==19.1.0
certifi==2019.3.9
chardet==3.0.4
Click==7.0
colorama==0.4.1
colorlog==4.0.2
dataclasses==0.6
decorator==4.3.2
entrypoints==0.3
flake8==3.7.7
flake8-polyfill==1.0.2
gitdb2==2.0.5
GitPython==2.1.11
idna==2.8
ipython-genutils==0.2.0
jsonschema==3.0.1
jupyter-core==4.4.0
mando==0.6.4
mccabe==0.6.1
nbformat==4.4.0
pkg-resources==0.0.0
plotly==3.7.0
progress==1.5
pycodestyle==2.5.0
pyflakes==2.1.1
pyrsistent==0.14.11
pytz==2018.9
radon==3.0.1
requests==2.21.0
retrying==1.3.3
six==1.12.0
smmap2==2.0.5
tabulate==0.8.3
traitlets==4.3.2
urllib3==1.24.1
wily==1.12.1

**Comment by tonybaloney** [comment]
thanks @mprostock I’ll try and reproduce this. have you tried wily on a couple of other projects?
Could you also share what your `locale` output is in bash

**Comment by tonybaloney** [comment]
theres actually an error handling condition that got merged for the halstead module but not published yet #63 which should at least skip this issue, report it on the screen and continue the build. I need to release that patch

**Comment by tonybaloney** [comment]
Released #63 in 1.12.2. Please could you update and try again

**Comment by mprostock** [comment]
Hey, thanks for the super quick response and fix and very sorry for my late reply!!

I check same repo with 1.12.2 and the core error is gone. It still complains about the same file but the indexing process continues and I can later use wily perfectly. So thanks for the fix. 

As I am not a wily expert, I find it a bit confusing that it keeps repeating the same error messages over and over (even while I guess it seems to be scanning other files), but as I said, the repo is indexed and I can use wily fine afterwards. Example (still same file as in my original report, and yes, there is an invalid character there (was there several commits ago, has been corrected long ago, but is of course still in the repo commit history)

![image](https://user-images.githubusercontent.com/12598309/54915098-b0676c00-4ef6-11e9-8d93-623b619b536a.png)
